ToBeRenamed.Tests/Commands/CreateLibraryHandlerTest.cs

```diff
@@ -0,0 +1,66 @@
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Dapper;
+using Respawn;
+using ToBeRenamed.Commands;
+using ToBeRenamed.Dtos;
+using Xunit;
+
+namespace ToBeRenamed.Tests.Commands
+{
+    [Collection("Database collection")]
+    public class CreateLibraryHandlerTest
+    {
+        DatabaseFixture fixture;
+
+        public CreateLibraryHandlerTest(DatabaseFixture fixture)
+        {
+            this.fixture = fixture;
+        }
+        
+        private static Checkpoint checkpoint = new Checkpoint
+        {
+            SchemasToInclude = new[]
+            {
+                "plum",
+                "public"
+            },TablesToIgnore = new[]
+            {
+                "users"
+            },
+            DbAdapter = DbAdapter.Postgres
+            
+        };
+            
+        [Fact]
+        public void Should_InsertIntoDB_When_PropertiesAreValid()
+        {
+            fixture.resetDatabase(checkpoint);
```

Comment:
`fixture.resetDatabase(checkpoint);` should be called before each test. I'm sure there's a way to do this automatically without manually calling it in each test, but I think this works temporarily for now